### PR TITLE
Check free space of quota

### PR DESCRIPTION
**Pull request by gabrielgisoldo**
Added code to check if the user has the necessary free space on his quota before making the upload. Otherwise it's unnecessary to start the upload.

### CODE REVIEW

**Comment by gabrielgisoldo** [comment]
Just resolved the conflict found after the last merge.

File: vimeo/exceptions.py [hunk 1]
```diff
@@ -36,8 +36,21 @@ def __init__(self, message):
         super(ObjectLoadFailure, self).__init__(message)
 
 
+class UploadQuotaExceeded(Exception):
+    """Exception for upload quota execeeded."""
+
+    def _get_free_space(self, num):
+        """Transform bytes in giga bytes."""
```
Comment (erunion) [inline vimeo/exceptions.py]: "gigabytes"

**Comment by erunion** [comment]
Tagged this to 0.4.1. Thank you!

**Comment by gabrielgisoldo** [comment]
No problem!

**Pull request merged**